modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -54,6 +54,12 @@ def _read(cls, filepath_or_buffer, **kwargs):
         # Ensures that the file is a string file path. Otherwise, default to pandas.
         filepath_or_buffer = cls.get_path_or_buffer(filepath_or_buffer)
         if isinstance(filepath_or_buffer, str):
+            is_folder = filepath_or_buffer.endswith("/")
+            if "*" not in filepath_or_buffer and not is_folder:
+                warnings.warn(
+                    "Shell-style wildcard '*' must be in the filename in order to read multiple "
+                    f"files at once; passed filename: '{filepath_or_buffer}'"
```

Comment:
```suggestion
                    "Shell-style wildcard '*' must be in the filename pattern in order to read multiple "
                    f"files at once. did you forget it? Passed filename: '{filepath_or_buffer}'"
```
so the user can check their code